R/activity_event.R

```diff
@@ -15,8 +15,6 @@
 #'
 #' @field activity  character [optional]
 #'
-#' @field data  \link{object} [optional]
```

Comment:
I think we should put this back, but it's actually called `static_data` -- there was an error in generating the code here.